src/main/java/org/jabref/gui/groups/GroupAddRemoveDialog.java

```diff
@@ -166,7 +167,8 @@ private boolean doAddOrRemove() {
             GroupTreeNodeViewModel node = (GroupTreeNodeViewModel) path.getLastPathComponent();
             if (checkGroupEnable(node)) {
 
-                List<BibEntry> entries = Globals.stateManager.getSelectedEntries();
```

Comment:
Probably need to use `entries` also below in https://github.com/JabRef/jabref/pull/4457/files#diff-9d27d25ddef48b30f2e1e1bd9e5de176R180